etc/adminhtml/di.xml

```diff
@@ -34,4 +34,64 @@
             <argument name="list" xsi:type="object">RedChamps\CleanMenu\Model\CustomTabList</argument>
         </arguments>
     </virtualType>
+    <type name="RedChamps\CleanMenu\Model\Config\Source\Rules">
+        <arguments>
+            <argument name="options" xsi:type="array">
+                <item name="allowedByDefault" xsi:type="array">
+                    <item name="value" xsi:type="const">RedChamps\CleanMenu\Model\AllowedRule\AllowedByDefault::RULE_CODE</item>
+                    <item name="label" xsi:type="string" translatable="true">Allowed by default (selected items are denied)</item>
+                </item>
+                <item name="deniedByDefault" xsi:type="array">
+                    <item name="value" xsi:type="const">RedChamps\CleanMenu\Model\AllowedRule\DeniedByDefault::RULE_CODE</item>
+                    <item name="label" xsi:type="string" translatable="true">Denied by default (selected items are allowed)</item>
+                </item>
```

Comment:
What about using this as the label "Selected items are allowed"? instead of "Denied by default (selected items are allowed)"